cms/models.py

```diff
@@ -0,0 +1,44 @@
+"""
+Page models for the CMS
+"""
+from django.db import models
+
+from wagtail.core.models import Page
+from wagtail.admin.edit_handlers import FieldPanel
+from wagtail.core.fields import RichTextField
+from wagtail.images.models import Image
+
+
+class ProgramPage(Page):
+    """
+    CMS page representing the a Program
+    """
+    description = RichTextField(
```

Comment:
Does the RichTextField allow for arbitrary HTML? The CMS users for mitxpro will be comfortable writing HTML and will want the added flexibility.